src/boot/bootctl.c

```diff
@@ -1022,6 +1035,12 @@ static int install_variables(const char *esp_path,
         uint16_t slot;
         int r;
 
+        if (arg_root) {
+                log_warning("Acting on external %s, skipping EFI variable setup.",
```

Comment:
"external"? not sure what is "external" about that?

I think this should be downgraded a bit, it's not a warning. It's notice or even info